File: основы16.09/main.py
#1
"""
a = int(input("Введите первое число: "))
b = int(input("Введите второе число: "))

if(a > b):
    print(f"{a} больше {b}")
elif(b > a):
    print(f"{b} больше {a}")
elif(a == b):
    print(f"Оба числа равные")
else:
    print("Неккоректный ввод")
"""

#2
"""
import math

a = int(input("Введите значение a: "))
b = int(input("Введите значение b: "))
c = int(input("Введите значение c: "))

d = b**2 - 4 * a * c

if (d > 0):
    x_1 = (-b + math.sqrt(d)) / (2 * a)  
    x_2 = (-b - math.sqrt(d)) / (2 * a)
    print(f"x_1 = {x_1}, x_2 = {x_2}")
elif (d == 0):
    x = (-b) / (2 * a)
    print(f"x = {x}")
elif (d < 0):
    print(f"Нету решения")
"""

#3
"""
year = int(input("Введите год: "))

if (year % 4 == 0 and (year % 100 != 0 or year % 400 == 0)):
    print(f"Год високосный")
else:
    print(f"Год невисокосный")
"""

#4
"""
n = int(input("Введите номер месяца: "))
if (n == 1):
    print(f"Месяц: Январь \n Кол-во дней: 31")
elif (n == 2):
    print(f"Месяц: Февраль \n Кол-во дней: 28")
elif (n == 3):
    print(f"Месяц: Март \n Кол-во дней: 31")
elif (n == 4):
    print(f"Месяц: Апрель \n Кол-во дней: 30")
elif (n == 5):
    print(f"Месяц: Май \n Кол-во дней: 31")
elif (n == 6):
    print(f"Месяц: Июнь \n Кол-во дней: 30")
elif (n == 7):
    print(f"Месяц: Июль \n Кол-во дней: 31")
elif (n == 8):
    print(f"Месяц: Август \n Кол-во дней: 31")
elif (n == 9):
    print(f"Месяц: Сентябрь \n Кол-во дней: 30")
elif (n == 10):
    print(f"Месяц: Октябрь \n Кол-во дней: 31")
elif (n == 11):
    print(f"Месяц: Ноябрь \n Кол-во дней: 30")
elif (n == 12):
    print(f"Месяц: Декабрь \n Кол-во дней: 31")
else:
    print(f"Неккоректный ввод")
"""

#5
"""
x = int(input("Введите первое число: "))
y = int(input("Введите второе число: "))
a = int(input("Выберите действие: 1.+   2.-   3.*   4./: "))

if (a == 1):
    print(f"{x} + {y} = {x + y}")
elif (a == 2):
    print(f"{x} - {y} = {x - y}")
elif (a == 3):
    print(f"{x} * {y} = {x * y}")
elif (a == 4):
    print(f"{x} / {y} = {x / y}")
else:
    print(f"Неккоректный ввод")
"""

#6
"""
sum = int(input("Введите сумму покупки: "))

if(sum > 1000):
    total_sum = (sum * 0.9)
    print(f"Итоговая сумма: {sum + total_sum}")
elif(sum > 500):
    total_sum = sum * 0.8
    print(f"Итоговая сумма: {sum + total_sum}")
else:
    print(f"Итоговая сумма: {sum}")
"""

#7
"""
x = int(input("Введите x: "))
y = int(input("Введите y: "))

if (x > 0 and y > 0):
    print("1 четверть")
elif (x < 0 and y > 0):
    print("2 четверть")
elif (x < 0 and y < 0):
    print("3 четверть")
elif (x > 0 and y < 0):
    print("4 четверть")
else:
    print("Неккоректный ввод")
"""

#8
"""
sum = int(input("Введите сумму вклада: "))
times = int(input("Введите срок в месяцах: "))

if (times < 6):
    total_sum = sum + (sum * 0.05)
    print(f"Итоговая сумма: {total_sum}")
elif (times >= 6 and times < 12):
    total_sum = sum + (sum * 0.07)
    print(f"Итоговая сумма: {total_sum}")
elif (times >= 12):
    total_sum = sum + (sum * 0.1)
    print(f"Итоговая сумма: {total_sum}")
else:
    print("Неккоректный ввод")
"""

File: основы16.09/main.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    setlocale(LC_ALL, "ru");

    /*
    1

    int a;
    int b;

    cout << "Введите первое число: ";
    cin >> a;
    cout << "Введите второе число: ";
    cin >> b;

    if (a > b) {
        cout << a << "больше" << b;
    }
    else if (b > a) {
        cout << b << "больше" << a;
    }
    else if (a = b) {
        cout << "Числа равные";
    }
    else {
        cout << "Неккоректный ввод";
    }
    */

    /*
    2

    int a, b, c;
    cout << "Введите значение a: ";
    cin >> a;
    cout << "Введите значение b: ";
    cin >> b;
    cout << "Введите значение c: ";
    cin >> c;

    int d = pow(b, 2) - 4 * a * c;

    if (d > 0) {
        float x_1 = (-b + sqrt(d)) / (2 * a);
        float x_2 = (-b - sqrt(d)) / (2 * a);
        cout << "x_1 = " << x_1;
        cout << "x_2 = " << x_2;
    }
    else if (d == 0) {
        float x = (-b) / (2 * a);
        cout << "x = " << x;
    }
    else if (d < 0) {
        cout << "Нету решения";
    }
    */

    /*
    3

    int year;
    cout << "Введите год: ";
    cin >> year;

    if (year % 4 == 0 and (year % 100 != 0 || year % 400 == 0)) {
        cout << "Високосный";
    }
    else {
        cout << "Невисокосный";
    }
    */

    /*
    4

    int n;
    cout << "Введите номер месяца: ";
    cin >> n;

    if (n == 1) {
        cout << "Год: Январь, Кол-во дней: 31";
    }
    else if (n == 2) {
        cout << "Год: Февраль, Кол-во дней: 28";
    }
    else if (n == 3) {
        cout << "Год: Март, Кол-во дней: 31";
    }
    else if (n == 4) {
        cout << "Год: Апрель, Кол-во дней: 30";
    }
    else if (n == 5) {
        cout << "Год: Май, Кол-во дней: 31";
    }
    else if (n == 6) {
        cout << "Год: Июнь, Кол-во дней: 30";
    }
    else if (n == 7) {
        cout << "Год: Июль, Кол-во дней: 31";
    }
    else if (n == 8) {
        cout << "Год: Август, Кол-во дней: 31";
    }
    else if (n == 9) {
        cout << "Год: Сентябрь, Кол-во дней: 30";
    }
    else if (n == 10) {
        cout << "Год: Октябрь, Кол-во дней: 31";
    }
    else if (n == 11) {
        cout << "Год: Ноябрь, Кол-во дней: 30";
    }
    else if (n == 12) {
        cout << "Год: Декабрь, Кол-во дней: 31";
    }
    else {
        cout << "Неккоректный ввод";
    }
    */

    /*
    5


    int x, y, a;

    cout << "Введите первое число: ";
    cin >> x;
    cout << "Введите второе число: ";
    cin >> y;
    cout << "Выберите действие: 1.+   2.-   3.*   4./:";
    cin >> a;

    if (a == 1) {
        cout << x << " + " << y << " = " << x + y;
    }
    else if (a == 2) {
        cout << x << " - " << y << " = " << x - y;
    }
    else if (a == 3) {
        cout << x << " * " << y << " = " << x * y;
    }
    else if (a == 4) {
        cout << x << " / " << y << " = " << x / y;
    }
    else {
        cout << "Неккоректный ввод";
    }
    */

    /*
    6

    int sum;
    cout << "Введите стоимость: ";
    cin >> sum;

    if (sum > 1000) {
        cout << "Итоговая стоимость: " << sum + (sum * 0.9);
    }
    else if (sum > 500) {
        cout << "Итоговая стоимость: " << sum + (sum * 0.8);
    }
    else {
        cout << "Итоговая стоимость: " << sum;
    }
    */

    /*
    7

    int x, y;
    cout << "Введите x: ";
    cin >> x;
    cout << "Введите y: ";
    cin >> y;

    if (x > 0 and y > 0) {
        cout << "1 четверть";
    }
    else if (x < 0 and y > 0) {
        cout << "2 четверть";
    }
    else if (x < 0 and y < 0) {
        cout << "3 четверть";
    }
    else if (x > 0 and y < 0) {
        cout << "4 четверть";
    }
    else {
        cout << "Неккоректный ввод"
    }
    */

    /*
    8
  
    int sum, times;
    cout << "Введите сумму вклада: ";
    cin >> sum;
    cout << "Введите срок в месяцах: ";
    cin >> times;

    if (times < 6) {
        cout << "Итоговая сумма: " << sum + (sum * 0.05);
    }
    else if (times >= 6 and times < 12) {
        cout << "Итоговая сумма: " << sum + (sum * 0.07);
    }
    else if (times >= 12) {
        cout << "Итоговая сумма: " << sum + (sum * 0.1);
    }
    else {
        cout << "Неккоректный ввод";
    }
    */
}